samples/bluetooth/observer/README.rst

```diff
@@ -27,4 +27,4 @@ Building and Running
 This sample can be found under :zephyr_file:`samples/bluetooth/observer` in the
 Zephyr tree.
 
-See :ref:`Bluetooth samples section <bluetooth-samples>` for details.
+See :zephyr:code-sample-category:`Bluetooth samples section <bluetooth>` for details.
```

Comment:
```suggestion
See :zephyr:code-sample-category:`bluetooth` samples for details.
```

Suggest to make this consistent with the others

Reply:
Sure! As you can probably imagine, I went for the easy "search and replace" without necessarily putting more thought into it  :)